server/openshift-template.yml

```diff
@@ -0,0 +1,338 @@
+      generation: null
+      importPolicy: {}
+      name: "8"
+      referencePolicy:
+        type: ""
+  status:
+    dockerImageRepository: ""
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    creationTimestamp: null
+    labels:
+      app: ionic-showcase-server
+    name: ionic-showcase-server
+  spec:
+    lookupPolicy:
+      local: false
+  status:
+    dockerImageRepository: ""
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    creationTimestamp: null
+    labels:
+      app: ionic-showcase-server
+    name: ionic-showcase-server
+  spec:
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ionic-showcase-server:latest
+    postCommit: {}
+    resources: {}
+    source:
+      contextDir: server
+      git:
+        uri: https://github.com/aerogear/ionic-showcase
+      type: Git
+    strategy:
+      dockerStrategy:
+        from:
+          kind: ImageStreamTag
+          name: node:8
+      type: Docker
+    triggers:
+    - github:
+        secret: 60MurnzAk_RzzjT3BYqy
+      type: GitHub
+    - generic:
+        secret: O30AOHys65x0WTxPgwaj
+      type: Generic
+    - type: ConfigChange
+    - imageChange: {}
+      type: ImageChange
+  status:
+    lastVersion: 0
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    creationTimestamp: null
+    labels:
+      app: ionic-showcase-server
+    name: ionic-showcase-server
+  spec:
+    replicas: 1
+    selector:
+      app: ionic-showcase-server
+      deploymentconfig: ionic-showcase-server
+    strategy:
+      resources: {}
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftNewApp
+        creationTimestamp: null
+        labels:
+          app: ionic-showcase-server
+          deploymentconfig: ionic-showcase-server
+      spec:
+        containers:
+        - env:
+            - name: DB_HOSTNAME
+              value: "${DATABASE_SERVICE_NAME}"
+          image: ionic-showcase-server:latest
+          name: ionic-showcase-server
+          ports:
+          - containerPort: 4000
+            protocol: TCP
+          resources: {}
+    test: false
+    triggers:
+    - type: ConfigChange
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - ionic-showcase-server
+        from:
+          kind: ImageStreamTag
+          name: ionic-showcase-server:latest
+      type: ImageChange
+  status:
+    availableReplicas: 0
+    latestVersion: 0
+    observedGeneration: 0
+    replicas: 0
+    unavailableReplicas: 0
+    updatedReplicas: 0
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      openshift.io/generated-by: OpenShiftNewApp
+    creationTimestamp: null
+    labels:
+      app: ionic-showcase-server
```

Comment:
the provisioned sync app is not showing up in the MDC. Can we add `mobile: enabled` label here so that it can show up in the MDC?